flink-contrib/flink-storm-compatibility/flink-storm-compatibility-core/src/main/java/org/apache/flink/stormcompatibility/api/FlinkTopology.java

```diff
@@ -28,17 +28,14 @@
  * cannot be executed directly, but must be handed over to a {@link FlinkLocalCluster}, {@link FlinkSubmitter}, or
  * {@link FlinkClient}.
  */
-class FlinkTopology extends StreamExecutionEnvironment {
+public class FlinkTopology extends StreamExecutionEnvironment {
```

Comment:
That was on purpose on no bug. Because, a user should on call `new FlinkTopology()`.

But now I need it to be public. Furthermore, and I am going to change some stuff and allow `new FlinkTopology()` (so I did not include a private constructor either in this PR -- for future purpose).